### PR TITLE
Add merchants resource

### PR DESCRIPTION
**Pull request by griffinmyers**
This PR adds support for the merchants resource.

A followup PR will cut a new minor version release.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -87,7 +87,75 @@ The supported options are as follows:
...
+
+Merchants
+~~~~~~~~~
+
+All
+'''
+
+You may pass the following optional arguments:
+
+* ``status`` (string): Status to filter by.  One of ('approved', 'pending', or 'available')
```
Comment (oritheorca) [inline README.rst]: Partnership request status, again

File: pybutton/resources/accounts.py [hunk 2]
```diff
@@ -7,22 +7,9 @@
...
-            timeout: The time in seconds for network requests to abort.
-              Defaults to None.
-              (N.B: Button's API is only exposed through HTTPS. This option is
-              provided purely as a convenience for testing and development.)
-
-    Raises:
-        pybutton.ButtonClientError
+    '''Manages interacting with Button Accounts via the Button API
+
+    See Resouce for class docstring.
```
Comment (oritheorca) [inline pybutton/resources/accounts.py]: Resource? Same typo in the below resources, as well.

**Review by oritheorca — APPROVED** [review]
Looks good!

File: pybutton/resources/accounts.py (hunk 2 above)
Comment (griffinmyers) [reply pybutton/resources/accounts.py]: 👍

**Comment by griffinmyers** [comment]
thank you, @gckwan !!

**Pull request merged**